requirements.txt

```diff
@@ -0,0 +1,33 @@
+absl-py==0.7.0
```

Comment:
How did you get all those deps? Eg do we use click somewhere?

Reply:
We don't use click but it installs automatically with Flask

Reply:
Why do we need it in requirements then?

Reply:
I reaplced it, done